en/docs/install-and-setup/perfromance-tuning-and-test-results/tuning-performance.md

```diff
@@ -414,15 +417,15 @@ The following parameters should be configured in the `<API-M_HOME>/repository/co
 [transport.binary.agent]
 :
 ```
-The Thrift section is related to Analytics and the Binary section is related to Throttling. Same set of parameters mentioned below can be found in both sections. The parameter descriptions and recommendations are intended towards the for performance tuning of Analytics, but the same recommendations are relevant for Throttling data related tuning in the Binary section. Note that the section for Thrift is relevant only if Analytics is enabled.
+The Thrift section is related to Analytics, and the Binary section is related to Throttling. The same set of parameters mentioned below can be found in both sections. The parameter descriptions and recommendations are intended towards the for performance tuning of Analytics, but the same recommendations are relevant for Throttling data related tuning in the Binary section. Note that the section for Thrift is relevant only if Analytics is enabled.
```

Comment:
The parameter descriptions and recommendations are intended towards the performance tuning of Analytics

Reply:
Done